src/renderer/renderElement.ts

```diff
@@ -38,6 +44,11 @@ const generateElementCanvas = (
   const canvas = document.createElement("canvas");
   const context = canvas.getContext("2d")!;
 
+  // To be able to draw a nested element with RTL, we have to append it to the DOM
+  canvas.style.display = "none";
+  canvas.id = "nested-canvas-element";
+  document.body.appendChild(canvas);
```

Comment:
This thing kills performance on moderate scenes. If anything, it should be done only for text elements, but I'll want to investigate if we can remove this altogether.